lib/workers/global/config/parse/index.ts

```diff
@@ -22,6 +22,19 @@ export async function parseConfigs(
   const cliConfig = cliParser.getConfig(argv);
   const envConfig = envParser.getConfig(env);
 
+  if (cliConfig?.dryRun === 'true') {
+    cliConfig.dryRun = 'full';
+  }
+  if (envConfig?.dryRun === 'true') {
+    envConfig.dryRun = 'full';
+  }
+  if (cliConfig?.dryRun === 'false') {
+    cliConfig.dryRun = null;
+  }
+  if (envConfig?.dryRun === 'false') {
+    envConfig.dryRun = null;
+  }
+
```

Comment:
I think we should log a warning here that the field values have changed, so that we can remove this code in a next major release